admin-api/create-cqrs-endpoints.md

```diff
@@ -0,0 +1,1023 @@
+---
+title: Create a new endpoint using CQRS
```

Comment:
```suggestion
title: Contribute to Core API endpoints using CQRS
```

Reply:
Original title is better for a developer documentation.

Reply:
We just discuss about it in meeting with @jolelievre. It's confusing bcz with "Create a new endpoint using CQRS", people will believe they can create a new endpoint on their shop, not contribute to Core endpoints.

But I let @jolelievre & you decide.